gateleen-delta/src/main/java/org/swisspush/gateleen/delta/DeltaHandler.java

```diff
@@ -200,6 +213,14 @@ private Long extractNumberDeltaParameter(String deltaStringId, HttpServerRequest
         }
     }
 
+    private void respondLimitOffsetParameterForbidden(HttpServerRequest request, Logger log){
+        String errorMsg = "limit/offset parameter not allowed for delta requests";
+        request.response().setStatusCode(StatusCode.BAD_REQUEST.getStatusCode());
+        request.response().setStatusMessage(StatusCode.BAD_REQUEST.getStatusMessage());
+        request.response().end(errorMsg);
+        log.warn(errorMsg);
```

Comment:
Possibly helpful to also log some trivial details to ease bug-hunting. Eg like:
```java
log.warn( "{}: {} {}", errorMsg, request.method(), request.uri());
```

Reply:
The used RequestLogger provides the unique request ID, which helps a lot in bug hunting.